### PR TITLE
Delete MANIFEST.in and .coveragerc from root directory

### PR DESCRIPTION
**Pull request by gsheni**
- Also move tests up one directory, and remove them from the released wheel + tar.gz file

### CODE REVIEW

File: docs/source/release_notes.rst [hunk 1]
```diff
@@ -9,6 +9,7 @@ Future Release
     * Changes
         * Transition to pure pyproject.toml for project metadata (:pr:`351`)
         * Change `target_dataframe_name` parameter name to `target_dataframe_index` (:pr:`353`)
+        * Delete MANIFEST.in from root directory (:pr:`359`)
```
Comment (gsheni) [inline docs/source/release_notes.rst:12]:
```suggestion
        * Delete MANIFEST.in and .coveragerc from root directory (:pr:`359`)
```

**Review by thehomebrewnerd — COMMENTED** [review]
I think the latest dependency checks are still trying to use the deleted `.coveragerc` file: https://github.com/alteryx/compose/blob/d7b592158f1d8dca71ac47981fdfbc01a2ad558e/.github/workflows/unit_tests_with_latest_deps.yml#L38

**Review by thehomebrewnerd — DISMISSED** [review]

File: pyproject.toml [hunk 2]
```diff
@@ -94,6 +94,10 @@ namespaces = true
     "*.txt",
     "README.md",
 ]
+"composeml.demos" = [
+    "*.csv",
+    "*.txt",
```
Comment (gsheni) [inline pyproject.toml:99]:
```suggestion
```

File: release.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ The environment variables `PYPI_USERNAME` and `PYPI_PASSWORD` must be already se
 
 ## Create Release Branch
 1. Branch off of main and name the branch the release version number (e.g. `release_v0.1.2`)
-2. Bump version number in `composeml/version.py`, and `composeml/tests/test_version.py`.
+2. Bump version number in `composeml/version.py`, and `tests/test_version.py`.
```
Comment (gsheni) [inline release.md:7]:
```suggestion
2. Bump version number in `composeml/version.py`, and `composeml/tests/test_version.py`.
```

**Review by thehomebrewnerd — APPROVED** [review]

**Pull request merged**